config/routes.yaml

```diff
@@ -1,3 +1,15 @@
 controllers:
     resource: ../src/Controller/
     type: attribute
+
+company-show:
+    path: /company/show
+    controller: App\Controller\CompanyController::show
+
+contact-show:
+    path: /contact/show
+    controller: App\Controller\ContactController::show
+
+home-show:
```

Comment:
use "_" instead of "-" for yaml variables. It works ok with "-" but It is a recommended best practice.